MLlib/activations.py

```diff
@@ -3,3 +3,34 @@
 
 def sigmoid(X):
     return 1/(1+np.exp(-X))
+
+
+def tanh(X):
+    return np.tanh(X)
+
+
+def softmax(X):
+    Sum = np.sum(np.exp(X))
+    return np.exp(X)/Sum
+
+
+def softsign(X):
+    return X/(np.abs(X) + 1)
+
+
+def relu(X):
+    return np.maximum(0,X)
+
+
+def leakyRelu(X):
+    return np.maximum(0.01*X,X)
+
+
+def elu(X):
+    output = []
```

Comment:
Why array is added?

Reply:
I couldn't figure out a way to correctly implement ELU in a single line
and We get a NumPy array as an input and also we need a NumPy array after applying activation as an output
So I created an empty list, calculated value for each element individually, and appended them to the list
And finally converted the list into NumPy array

Reply:
With reference to https://ml-cheatsheet.readthedocs.io/en/latest/activation_functions.html#:~:text=delta(x)%20!-,ELU,which%20should%20be%20positive%20number.&text=They%20are%20both%20in%20identity%20function%20form%20for%20non%2Dnegative%20inputs. ,
instead of array woudn't this work,

```
if x>0:
    return x
else:
    return (alpha*(np.exp(x)-1))
```

Reply:
I don't think so
I tried it but couldn't get the correct result

Reply:
Can you reploduce the code in the section, it works fine for me.